examples/places-utils/src/places-utils.rs

```diff
@@ -43,127 +32,8 @@ struct DesktopItem {
     children: Vec<DesktopItem>,
 }
 
-fn convert_node(dm: DesktopItem) -> Option<BookmarkTreeNode> {
```

Comment:
I'd rather not lose this code if we can avoid it. It certainly doesn't need to be interruptable though.